applications/FluidDynamicsApplication/custom_processes/distance_smoothing_process.cpp

```diff
@@ -122,7 +122,7 @@ void DistanceSmoothingProcess<TDim, TSparseSpace, TDenseSpace, TLinearSolver>::E
 
             auto& n_nodes = rNode.GetValue(NEIGHBOUR_NODES);
             for (unsigned int j = 0; j < n_nodes.size(); ++j) {
-                if (!(n_nodes[j].Is(CONTACT)) || !(rNode.Is(CONTACT))){
+                if ((n_nodes[j].Is(CONTACT) && rNode.Is(CONTACT)) || !(rNode.Is(CONTACT))){
```

Comment:
As far as I see this implies a behaviour change. Next time please try to encapsulate it in a single PR to ease keeping track of the changes.